docs/en/guides/developer/working-with-json/json-load-data.md

```diff
@@ -0,0 +1,147 @@
+- Examine the file format by selecting one row using the S3 function
+- Create a table to store the data in ClickHouse
+- Load a single row of nested JSON
+- Verify the correct storage of the nested JSON
+- Import the dataset from S3
+
+## Examine the structure of the JSON file
+Examine the structure and one record from the log file in S3.  The `s3` function
+retrieves and decompresses the file and allows querying the file
+in S3 without loading it.
+
+#### DESCRIBE
+```sql
+DESCRIBE s3('https://datasets-documentation.s3.eu-west-3.amazonaws.com/http/documents-01.ndjson.gz', 
+'JSONEachRow');
+```
+```response
+┌─name───────┬─type──────────────────────────┬─default_type─┬─default_expression─┬─comment─┬─codec_expression─┬─ttl_expression─┐
+│ @timestamp │ Nullable(Int64)               │              │                    │         │                  │                │
+│ clientip   │ Nullable(String)              │              │                    │         │                  │                │
+│ request    │ Map(String, Nullable(String)) │              │                    │         │                  │                │
+│ status     │ Nullable(Int64)               │              │                    │         │                  │                │
+│ size       │ Nullable(Int64)               │              │                    │         │                  │                │
+└────────────┴───────────────────────────────┴──────────────┴────────────────────┴─────────┴──────────────────┴────────────────┘
+```
+
+#### SELECT
+```sql
+SELECT * FROM s3('https://datasets-documentation.s3.eu-west-3.amazonaws.com/http/documents-01.ndjson.gz', 
+'JSONEachRow') LIMIT 1;
+```
+```response
+┌─@timestamp─┬─clientip───┬─request──────────────────────────────────────────────────────────┬─status─┬──size─┐
+│  893964617 │ 40.135.0.0 │ {'method':'GET','path':'/images/hm_bg.jpg','version':'HTTP/1.0'} │    200 │ 24736 │
+└────────────┴────────────┴──────────────────────────────────────────────────────────────────┴────────┴───────┘
+```
+Note that the `response` field contains nested JSON, it is more
+efficient for the users of the log data if that JSON is also extracted
+into separate fields. The next two steps will be performed with this
+optimization in mind.
+
+## Create a ClickHouse table
+To maximize the usefulness of the data we
+need to extract the nested `method`, `path`, and `version` fields under `request`.  To prepare for this, create a table including those nested fields:
+```sql
+CREATE table http
+(
+   timestamp    DateTime,
+   clientip     IPv4,
+# highlight-next-line
+   request Nested(method LowCardinality(String), path String, version LowCardinality(String)),
+   status       UInt16,
+   size         UInt32
+) ENGINE = MergeTree() ORDER BY (status, timestamp);
+```
+
+## Insert one row
+```sql
+INSERT INTO http (timestamp, clientip, request.method, request.path, request.version, status, size) SELECT
```

Comment:
I think using list of columns to insert is unnececcary here because we have all columns in the select query in right order

Reply:
yes, I bet that was in there because of the ephemeral before, I will remove it

Reply:
works fine, removed.